src/App.scss

```diff
@@ -1 +1,52 @@
-// styles go here
+.good {
+  &--red {
+    color: red;
+  }
+
+  &--green {
+    color: green;
+  }
+
+  &--blue {
+    color: blue;
+  }
+}
+
+.error {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  border-radius: 3px;
+
+  width: 200px;
+  height: 50px;
+
+  color: #d8000c;
+  background-color: #faadad;
+}
+
+.spinner {
+  display: block;
+  width: 80px;
+  height: 80px;
+}
+.spinner:after {
+  content: " ";
+  display: block;
+  width: 64px;
+  height: 64px;
+  margin: 8px;
+  border-radius: 50%;
+  border: 6px solid #a7a7a7;
+  border-color: #a7a7a7 transparent #a7a7a7 transparent;
+  animation: spinner 1.2s linear infinite;
+}
```

Comment:
Use scss nesting, like `&:after`

Reply:
Also it'd be better to use two columns version of after (`::after`) as it is more convenient